image_segmentation:
  ros__parameters:
    frozen_graph: /home/rosuser/ws/ros2_ws/src/image_segmentation_r2/models/mobilenet_v3_small_968_608_os8.pb
    xml_conversion_file: /home/rosuser/ws/ros2_ws/src/image_segmentation_r2/models/convert_cityscapes_to_ika_reduced.xml
    resize_width: 968
    resize_height: 608
